task8/{UPMessenger}/src/main/java/Posts.java

```diff
@@ -0,0 +1,71 @@
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+
+public class Posts {
+
+
+    public static List<Post> getPosts() {
+        return posts;
+    }
+
+    public static void setPosts(List<Post> posts) {
+        Posts.posts = posts;
+    }
+
+    private static List<Post> posts = new ArrayList<>();
+
+    Posts(){
```

Comment:
Why are you initializing in **instance constructor** a *static* field that has already been initialized as static? I may be missing something, or else this is wrong. Static fields are *one per class* they are actually initialized *before* an instance is initialized. And if you are writing something to a stati field, you are actually spoiling all the instances, as many as you have